packages/react-integration/demo-app-ts/src/components/demos/TopologyDemo/components/StyleNode.tsx

```diff
@@ -32,7 +32,7 @@ const ICON_PADDING = 20;
 type StyleNodeProps = {
   element: Node;
   getCustomShape?: (node: Node) => React.FC<ShapeProps>;
-  getShapeDecoratorCenter?: (quadrant: TopologyQuadrant, node: Node, radius?: number) => { x: number; y: number };
+  getShapeDecoratorCenter?: (quadrant: TopologyQuadrant, node: Node) => { x: number; y: number };
```

Comment:
Is it possible this is a change that could break some products?

Reply:
I don't think so, tit wasn't being used at all so I can't see why anyone would have passed it. If they did, it will be caught at compile time so shouldn't be a big issue.